docs/advanced/cmsis-rtos.md

```diff
@@ -1,77 +1,76 @@
+Therefore developers need to use the mbed OS RTOS C++ API or supply backing memory for RTX objects to `os*New` calls when using CMSIS-RTOS2 APIs directly. (Please consult CMSIS-RTOS2 documentation for API details.) `mbed_rtos_storage.h` header provides handy wrappers that you can use to secure required memory without exposing the code to RTX implementation details.
 
 ## Configuration
 
-mbed OS changes to RTX configuration are all maintained in a single file: `mbed-os/rtos/rtx2/mbed_rtx_conf.h`
+mbed OS changes to RTX configuration all exist in a single file: `mbed-os/rtos/rtx2/mbed_rtx_conf.h`
 
 Option | Value | Description |
 -------|-------|-------------|
-OS_STACK_SIZE | 4K or 2K | For normal target the thread stack size is set to 4K for constrained targets it's 2K |
-OS_TIMER_THREAD_STACK_SIZE | 768B | Timer thread stack set to 768B that's necessary to support our C++ wrappers (4 instances), but may require changing to support larger number of active timers |
-OS_IDLE_THREAD_STACK_SIZE | 256B | Required to handle mbed OS wrappers |
-OS_DYNAMIC_MEM_SIZE | 0 | RTX dynamic memory is disabled |
-OS_MUTEX_OBJ_MEM | 1 or 0 | For ARMC we use 1, for other toolchains it's 0. ARMC uses statically allocated mutexes internally. |
-OS_MUTEX_NUM | 6 or 0 | For ARMC we use 6, for other toolchains it's 0. ARMC uses statically allocated mutexes internally. |
-OS_STACK_WATERMARK | 0 or 1 | Watermarking enabled if MBED_STACK_STATS_ENABLED or MBED_STACK_STATS_ENABLED are set. |
-OS_PRIVILEGE_MODE | 0 or 1 | We set it for 0 if uVisor is enabled, 1 otherwise. |
-
+`OS_STACK_SIZE` | 4K or 2K | For a normal target, the thread stack size is set to 4K; for constrained targets, it's 2K. |
+`OS_TIMER_THREAD_STACK_SIZE` | 768B | Timer thread stack set to 768B that's necessary to support the C++ wrappers (4 instances), but it may require changing to support larger number of active timers. |
+`OS_IDLE_THREAD_STACK_SIZE` | 256B | Required to handle mbed OS wrappers |
+`OS_DYNAMIC_MEM_SIZE` | 0 | RTX dynamic memory is disabled. |
+`OS_MUTEX_OBJ_MEM` | 1 or 0 | For ARMC, use 1; for other toolchains, it's 0. ARMC uses statically allocated mutexes internally. |
+`OS_MUTEX_NUM` | 6 or 0 | For ARMC, use 6; for other toolchains, it's 0. ARMC uses statically allocated mutexes internally. |
+`OS_STACK_WATERMARK` | 0 or 1 | Watermarking is enabled if `MBED_STACK_STATS_ENABLED` or `MBED_STACK_STATS_ENABLED` are set. |
+`OS_PRIVILEGE_MODE` | 0 or 1 | We set it for 0 if uVisor is enabled, 1 otherwise. |
 
 ## Code structure
 
-Due to differences in how mbed OS and CMSIS directory structures look like, we can't import the original code directly, some directory changes are necessary:
+Due to differences in how the mbed OS and CMSIS directory structures look, you can't import the original code directly. Some directory changes are necessary:
```

Comment:
That's not a big deal, but it's not about the users, it's about the maintainers (us). This import is done, and may be needed to be done in the future to update CMSIS (external component) version.